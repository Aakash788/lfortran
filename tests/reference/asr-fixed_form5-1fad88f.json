{
    "basename": "asr-fixed_form5-1fad88f",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form5.f",
    "infile_hash": "620d881af257eb1c53b26d138f8f5ba7bbb589226cfe8a9bc7ec351b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form5-1fad88f.stdout",
    "stdout_hash": "d744df275a9fab3840ac4430ea06cbe4ac7d7e5581d42ddfc89d3920",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}