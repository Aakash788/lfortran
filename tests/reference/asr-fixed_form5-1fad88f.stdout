(TranslationUnit
    (SymbolTable
        1
        {
            zdotc:
                (Function
                    (SymbolTable
                        2
                        {
                            n:
                                (Variable
                                    2
                                    n
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            zdotc:
                                (Variable
                                    2
                                    zdotc
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Complex 8 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            zx:
                                (Variable
                                    2
                                    zx
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Complex 8 [(()
                                    ())])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    zdotc
                    []
                    [(Var 2 n)
                    (Var 2 zx)]
                    [(Return)]
                    (Var 2 zdotc)
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                )
        })
    []
)
